contents/docs/product-analytics/ai.md

```diff
@@ -0,0 +1,41 @@
+---
+title: AI and LLM insights (beta)
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+> This integration is currently in private beta. If you'd like to be added to beta, email Lior@posthog.com.
+
+We've teamed up with [Langfuse](https://langfuse.com/) to track metrics for products that use LLMs and AI models such as OpenAI, Langchain, and LlamaIndex. Our integration monitors metrics such as generation latency, model costs, usage count, and more. It makes it easy to answer questions like:
```

Comment:
None of these are LLMs themselves, more like tools and model providers for LLM applications. 

Maybe I'm misunderstanding what LangChain and LlamaIndex do, but they seem like frameworks for using LLMs and I'm not sure what analytics we would capture from them?

```suggestion
We've teamed up with [Langfuse](https://langfuse.com/docs/analytics/posthog) to track metrics for LLM applications. Our integration monitors metrics such as generation latency, model costs, usage count, and more. It makes it easy to answer questions like:
```